### PR TITLE
Admin user

### PR DESCRIPTION
**Pull request by thanh-cao**
Issues fixed: 
#21 user able to create new organisation and edit
#20 user able to delete an organisation from database
#19 a table listing all organisations available with buttons to edit and delete

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!--Materialize CDN-->
+	<!-- Compiled and minified CSS -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```
Comment (auxfuse) [inline templates/base.html:10]: We are using Bootstrap 5 for this project.

Can you update this to reflect, and then match the utility class usage in the template files to suit Bootstrap 5 please? 😀

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		{% block content %}
+		{% endblock %}
+	</main>
+	<!--./end-main-content-->
+	<!--Start Footer-->
+	<footer></footer>
+	<!--./end-footer-->
+	
+	<!-- Compiled and minified JavaScript -->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```
Comment (auxfuse) [inline templates/base.html:38]: As above, can this be removed to suit Bootstrap inclusion? 🤜

File: templates/create_organisation.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+{% block content %}
+
+<h3>Create Organization</h3>
+<form class="col s12" method="POST" action="{{ url_for('organisation_create') }}">
```
Comment (auxfuse) [inline templates/create_organisation.html:5]: This is an example of the utility classes that will need to be updated to suit Bootstrap usage over Materialize. Anywhere in these html files, that utilises Materialize Utility Classes will need to be updated to suit Bootstrap.

**Review by auxfuse — CHANGES_REQUESTED** [review]
Hi Thanh, thanks for the hard work gone into this. It's a big one! 🤜

The first thing we want to address is the Materialize usage. See comments throughout the PR. Once done, push those changes to the branch associated with this PR to auto-update same and then we can review again. 😀

**Comment by thanh-cao** [comment]
@auxfuse I have updated Materialize's cdn to Bootstrap5 and Bootstrap icons. Classes on all three templates for organisation's CRUD are also updated accordingly with bootstrap :)

**Comment by auxfuse** [comment]
Hi @thanh-cao thank you for the updates 😀We've just merged another PR that has unfortunately now given your's some conflicts in the `app.py` file. Can you resolve, push and then we can merge 💯

**Comment by thanh-cao** [comment]
@auxfuse I have fixed the conflicts in the app.py. Reason for conflict was that both John and I created same functionality to create new organisation BUT for 2 different user roles: admin and normal user. I have kept John's code and adjusted a mine accordingly to keep name consistency. I have left a todo note there for some discussion if we should define permission here for admin and user to be implemented later.

Made some adjustments to the templates to keep the styling consistent in both create and edit templates.
I didn't make any adjustments to company_web_scraper.py but for some reason it says on my branch that I have to commit the file so. I have checked with the content of the file on this repo and there isn't any difference between the two.

**Review by auxfuse — APPROVED** [review]
Everything looks great to me here! Well done @thanh-cao 🤜

**Review by auxfuse — APPROVED** [review]
Everything looks great to me here! Well done @thanh-cao 🤜

**Pull request merged**